content/asp.net/getting-started/data.md

```diff
@@ -1,20 +1,168 @@
 # Working with Data in your ASP.NET Core App
-by [Steve Smith](http://deviq.com/me/steve-smith)
+by [James Chambers](http://deviq.com/me/james-chambers)
 
 #### Sample Files
 Download a ZIP containing this tutorial's sample files:
-- [Initial Version] - Use this as a starting point when following along with the tutorial yourself
-- [Completed Version] - Includes the completed versions of all samples
+- [Completed Version (Pending)] - Includes the completed versions of all MVC tutorials
 
-See the [Issue](https://github.com/dotnet/training-tutorials/issues/67) to claim this lesson and/or view what should be included in it.
+To really start to add value to your application you're going to need to start loading, saving and displaying data. In ASP.NET Core MVC the tool of choice will likely be Entity Framework Core, or EF Core. In this example we're going to explore the idea of logging HTTP 404 errors to an in-memory database.
 
-## First Header
+## Entity Framework Core
 
-Start the lesson here.
+According to the documentation, [Entity Framework](https://docs.microsoft.com/en-us/ef/core/) is "an object-relational mapper (O/RM) that enables .NET developers to work with a database using .NET objects". The database you connect with doesn't really matter to your code and you don't have to worry about the details. In fact, you can use a relational data store such as SQL Server, or you can use an in-memory database or any other back-end and your code will largely remain the same. 
 
-...
+By using a pre-built library we can avoid having to write a lot of the data access code on our own.  We'll look at that using the in-memory database provider that EF Core supports. To do so, we need to first add a reference to the relevant libraries to our project. Right-click on your project and select "Manage NuGet Packages..." then install both `Microsoft.EntityFrameworkCore` and `Microsoft.EntityFrameworkCore.InMemory`.  The first package gives you all the primitives, base types and features of the OR/M, while the second gives you the ability to use an in-memory data store.
```

Comment:
Stick with "you" over "we". Also while it's probably good to mention both packages, I think if they just include 'Microsoft.EntityFrameworkCore.InMemory" it will pull in its dependency on 'Microsoft.EntityFrameworkCore' right? So they would only need to add one package at that point (maybe worth noting at the end of this paragraph).